Quaver.Shared/Config/ConfigManager.cs

```diff
@@ -806,16 +806,19 @@ public static void Initialize()
         /// </summary>
         private static void ReadConfigFile()
         {
-            try
-            {
-                // Delete the config file if we catch an exception.
-                var _ = new IniFileParser.IniFileParser(new ConcatenateDuplicatedKeysIniDataParser()).ReadFile(_gameDirectory + "/quaver.cfg")["Config"];
-            }
-            catch (ParsingException)
+            if (File.Exists(_gameDirectory + "/quaver.cfg"))
             {
-                Logger.Important("Config file couldn't be read.", LogType.Runtime);
-                File.Copy(_gameDirectory + "/quaver.cfg", _gameDirectory + "/quaver.corrupted." + TimeHelper.GetUnixTimestampMilliseconds() + ".cfg");
-                File.Delete(_gameDirectory + "/quaver.cfg");
+                try
+                {
+                    // Delete the config file if we catch an exception.
+                    var _ = new IniFileParser.IniFileParser(new ConcatenateDuplicatedKeysIniDataParser()).ReadFile(_gameDirectory + "/quaver.cfg")["Config"];
+                }
+                catch (ParsingException)
+                {
+                    Logger.Important("Config file couldn't be read.", LogType.Runtime);
+                    File.Copy(_gameDirectory + "/quaver.cfg", _gameDirectory + "/quaver.corrupted." + TimeHelper.GetUnixTimestampMilliseconds() + ".cfg");
+                    File.Delete(_gameDirectory + "/quaver.cfg");
```

Comment:
`_gameDirectory + "/quaver.cfg"` is repeated multiple times. Use a variable instead.